recipes/PeoplesSpeech/ASR/transformer/hparams/conformer_large.yaml

```diff
@@ -0,0 +1,287 @@
+# ############################################################################
+# Model: E2E ASR with Transformer
+# Encoder: Conformer Encoder
+# Decoder: Transformer Decoder + (CTC/ATT joint) beamsearch + TransformerLM
+# Tokens: BPE
+# losses: CTC + KLdiv (Label Smoothing loss)
+# Training: People's Speech 28,000 hours
+# Authors:  Titouan Parcollet
+# ############################################################################
+# Seed needs to be set at top of yaml, before objects with parameters are made
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 3407
+__set_seed: !apply:speechbrain.utils.seed_everything [!ref <seed>]
+output_folder: !ref results/conformer_transducer_large/<seed>
+output_wer_folder: !ref <output_folder>/
+save_folder: !ref <output_folder>/save
+train_csv: !ref <save_folder>/train.csv
+train_log: !ref <output_folder>/train_log.txt
+data_folder: /ephemeral/datasets/people_speech
+test_2: /fss/shared_saic-speech/t.parcollet/mnt/results/conformer_en/3407/test.csv
+
+# Data files
+hf_download_folder: !PLACEHOLDER
+subsets: ["dirty", "clean"] # _sa sets are removed.
+ckpt_interval_minutes: 5 # save checkpoint every N min
+
+
+####################### Training Parameters ####################################
+
+number_of_epochs: 30
+optimizer_step_limit: 400000
+ctc_weight: 0.3
+grad_accumulation_factor: 1
+max_grad_norm: 5.0
+loss_reduction: 'batchmean'
+sorting: random
+num_workers: 4
+precision: fp16 # bf16, fp16 or fp32
+skip_prep: False
+
+# stages related parameters
+lr_adam: 0.0008
+
+# Feature parameters
+sample_rate: 16000
+n_fft: 512
+n_mels: 80
+win_length: 32
+
+# This setup works well for A100 80GB GPU, adapts it to your needs.
+# Or turn it off (but training speed will decrease)
+max_batch_length_train: 500
+max_batch_length_val: 100 # we reduce it as the beam is much wider (VRAM)
+num_bucket: 200
+shuffle: True # if true re-creates batches at each epoch shuffling examples.
+batch_ordering: random
+max_batch_ex: 256
```

Comment:
I am wondering if we shouldn't talk about htat in the README?  I have always found people struggling with the concept of minimal audio hours per optimisation step. Would be maybe nice to say that you need to play with these parameters to fit them in your GPU while trying to reach N hours of audio per adam step

Reply:
No, we have a tutorial about that.